modules/migration-installing-cam-operator-ocp-3.adoc

```diff
@@ -91,7 +91,7 @@ containers:
 ...
     env:
     - name: REGISTRY
-      value: <registry.apps.example.com> <2>
+      value: <registry.apps.example.com> <3>
```

Comment:
@apinnick This PR is fixing build issues. Can you please check that this fix is correct? Thanks!

Reply:
Yes, this looks fine.